### ISSUE TITLE
When viewing Insights report, the select boxes are not wide enough for the whole text to be readable at once

### ISSUE THREAD

**Issue by mattab** [matomo-org/matomo#10562]
this is optional to fix, in case it's not trivial...
- When viewing Insights report, the select lists with the text `same day in the previous week` and `Increaser & decreaser` are truncated
- is it possible to make the select lists wider so the whole text is viewable?

![is it possible to make the select lists wider so the whole text is viewable](https://cloud.githubusercontent.com/assets/466765/18822677/045884ee-840f-11e6-9934-1b9858aa8f6c.png)

**Comment by tsteur** [comment]
Yeah I know about that one. This is pretty much only the case in the dashboard and only when the text is long. We can't do much here apart from making the fields full width alternatively. But it would then be also full width when you not view it in a dashboard but inside a report where it would use huge space

**Comment by mattab** [comment]
Q: could we maybe make the text within the select box, Wrapped, so it's fully readable?

**Comment by mattab** [comment]
This was slightly improved recently in https://github.com/matomo-org/matomo/pull/23733 and not much more can be done, so closing
